src/sdk/analytics/index.tsx

```diff
@@ -19,7 +26,17 @@ export const AnalyticsHandler = ({ children }: PropsWithChildren<unknown>) => {
     )
   })
 
-  return children
+  if (typeof window === 'undefined') {
+    return null
```

Comment:
```suggestion
    return children
```

When the component returns null all the subtree isn't rendered. So doing this, the SSG won't work as expected